Source/Extensions/Blazorise.ImageCropper/Enums/ImageSmoothingQuality.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Blazorise.ImageCropper;
+
+/// <summary>
+/// Cropped Image Smoothing Quality
+/// </summary>
+public enum ImageSmoothingQuality
```

Comment:
Rename to `ImageCropperImageSmoothingQuality`?

Reply:
ok